lib/file-list.js

```diff
@@ -62,9 +62,9 @@ class FileList {
 
     let lastCompletedRefresh = this._refreshing
     lastCompletedRefresh = Promise.all(
-      this._patterns.map(async ({ pattern, type, nocache, isBinary }) => {
+      this._patterns.map(async ({ pattern, type, nocache, isBinary, integrity }) => {
```

Comment:
What will the behavior be if a pattern matches multiple files? Will all the matched files have the same hash? If so, that makes no sense, right?

Reply:
Maybe we could validate that when integrity is used, a pattern cannot match more than a single file or validate that the value passed to pattern has no wildcards, which would yield the same effect.

Reply:
Integrity is verified only if a pattern is an absolute URL.

Reply:
You mean in the browser?

Reply:
That code doesn't seem to depend on environments.

Reply:
Right. Sorry!